File: flaskProject/alchemyClasses/Pelicula.py
from sqlalchemy import Column, Integer, String
from alchemyClasses import db


class Pelicula(db.Model):

    __tablename__ = 'peliculas'
    idPelicula = Column(Integer, primary_key=True)
    nombre = Column(String(200))
    genero = Column(String(200),nullable=True)
    duracion= Column(Integer, nullable=True,default=None)
    inventario=Column(Integer,default=1)

    def __init__(self, nombre, genero=None, duracion=None, inventario=1):
        self.nombre = nombre
        self.genero = genero
        self.duracion = duracion
        self.inventario = inventario
        
    def __str__(self):
        return f'\nNombre:{self.nombre}, Genero:{self.genero} \n'

File: my-app/src/components/Peliculas/Pelicula/Pelicula.js
import React from "react";

import './Pelicula.css';
 
const Pelicula = ({pelicula}) => {
    return (
        <div className='alumnos'>
            <div className="alumno__description">
                <h2>{pelicula.id}</h2>
                <h2>{pelicula.nombre}</h2>
                <h2>{pelicula.genero}</h2>
                <h2>{pelicula.duracion}</h2>
                <h2>{pelicula.inventario}</h2>
            </div>
        </div>
    );
}

export default Pelicula